lib/install/validate/validator.go

```diff
@@ -156,6 +157,30 @@ func NewValidator(ctx context.Context, input *data.Data) (*Validator, error) {
 	return v, nil
 }
 
+var schemeMatch = regexp.MustCompile(`^\w+://`)
```

Comment:
https://play.golang.org/p/mRrrzoOS1_ could work but I'm OK with the change if I'm the only one stuck on regex :)